src/index.ts

```diff
@@ -55,6 +52,8 @@ export {
   logger,
 };
 
+export const app = { setEmulatedAdminApp };
```

Comment:
sigh I wish I knew why this function is needed for the emulator to work.

(I have a theory that this is just a historical artifact before admin sdk natively supported the emulator)